packages/block-library/src/button/index.php

```diff
@@ -17,7 +17,15 @@
  * @return string The block content.
  */
 function render_block_core_button( $attributes, $content ) {
-	$p = new WP_HTML_Tag_Processor( $content );
+	/*
+	 * The current Gutenberg plugin supports WordPress 6.4, but the next_token()
+	 * method does not exist in WordPress 6.4. Therefore, use the Gutenberg
+	 * class that has the `next_token()` method.
+	 *
+	 * TODO: After the Gutenberg plugin drops support for WordPress 6.4, this
+	 * class should be replaced with `WP_HTML_Tag_Processor`.
+	 */
+	$p = new Gutenberg_HTML_Tag_Processor_6_5( $content ); // phpcs:ignore Gutenberg.CodeAnalysis.ForbiddenFunctionsAndClasses.ForbiddenClassUsage
```

Comment:
@t-hamano, I recommend using the [code](https://github.com/WordPress/gutenberg/pull/63082/files#r1664311551) from your own example, as `Gutenberg_HTML_Tag_Processor_6_5` isn't available in WordPress Core. This will help avoid any fatal errors when the code is backported to WordPress Core. Thanks. 
```suggestion
	if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN && class_exists( 'Gutenberg_HTML_Tag_Processor_6_5' ) ) {
	$p = new Gutenberg_HTML_Tag_Processor_6_5( $content );
} else {
	$p = new WP_HTML_Tag_Processor( $content );
}
```